Examples/WildeGuess/WildeGuess/Components/QuoteComponent.mm

```diff
@@ -19,10 +19,24 @@
 #import "WarmQuoteComponent.h"
 
 @implementation QuoteComponent
+{
+  Quote *_quote;
+  QuoteContext *_context;
+}
 
 + (instancetype)newWithQuote:(Quote *)quote context:(QuoteContext *)context
 {
-  return [super newWithComponent:quoteComponent(quote, context)];
+  auto const c = [super new];
+  if (c) {
+    c->_quote = quote;
+    c->_context = context;
+  }
+  return c;
+}
```

Comment:
I really dislike this. Can we not figure out a way to type the props and provide the generic initializer?

Reply:
I had the impression that you tried a year ago and you couldn't find a good solution to proceed with. We don't like it either, but I wouldn't make it as a blocker for the render change.